File: src/main/java/Example1.java
/**
 * Created by edutilos on 07.09.17.
 */
public class Example1 {
    public static void main(String[] args) {
         Person p1 , p2 ;
         p1 = new Person(1L, "foo", 10);
         p2 = new Person(2L, "bar", 20);
        System.out.println(p1);
        System.out.println(p2);
    }


    static class Person {
        private long id;
        private String name;
        private int age;

        public Person(long id, String name, int age) {
            this.id = id;
            this.name = name;
            this.age = age;
        }

        public Person() {

        }

        public long getId() {
            return id;
        }

        public void setId(long id) {
            this.id = id;
        }

        public String getName() {
            return name;
        }

        public void setName(String name) {
            this.name = name;
        }

        public int getAge() {
            return age;
        }

        public void setAge(int age) {
            this.age = age;
        }

        @Override
        public String toString() {
            return "Person{" +
                    "id=" + id +
                    ", name='" + name + '\'' +
                    ", age=" + age +
                    '}';
        }
    }
}

File: src/main/kotlin/com/edutilos/test1/Example1.kt
package com.edutilos.test1

/**
 * Created by edutilos on 07.09.17.
 */
object Example1 {
    @JvmStatic fun main(args: Array<String>) {
        val p1: Person
        val p2: Person
        p1 = Person(1L, "foo", 10)
        p2 = Person(2L, "bar", 20)
        println(p1)
        println(p2)
    }


    internal class Person {
        var id: Long = 0
        var name: String? = null
        var age: Int = 0

        constructor(id: Long, name: String, age: Int) {
            this.id = id
            this.name = name
            this.age = age
        }

        constructor() {

        }

        override fun toString(): String {
            return "Person{" +
                    "id=" + id +
                    ", name='" + name + '\'' +
                    ", age=" + age +
                    '}'
        }
    }
}